lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -130,7 +130,8 @@
         hasOwnProperty(o, prop) { return callInstanceFunc(platform.builtInJavascriptObjectEntryHasOwnProperty, o, prop); },
         defineProperty: platform.builtInJavascriptObjectEntryDefineProperty,
         isExtensible: platform.builtInJavascriptObjectEntryIsExtensible,
-        create() { return platform.builtInJavascriptObjectCreate(null); },
+        create(proto = null) { return platform.builtInJavascriptObjectCreate(proto); },
+        setPrototypeOf(target, proto = null) { return platform.builtInSetPrototype(target, proto); },
```

Comment:
I found this to be confusing; I saw an instance of `_.setPrototypeOf` and thought that it was the same as the js-spec `Object.setPrototypeOf` which has a required second argument, but had to look back up here to see that this had a default argument. I'd prefer if this was just a straight pass-through, and call-sites explicitly provide null if required.

Reply:
Null is what we want to set the prototype to 99% of the time in Intl.js. I have a few such default arguments in the _ object. I was finding providing a second argument to create a lot of code noise.

Reply:
_Interesting side note:_ platform.builtInSetPrototype is not the same function as Object.setPrototypeOf, and I am not entirely clear as to why.

Reply:
^ Maybe add a comment to that effect so we don't lose track of that bit of trivia -- future maintainer to update the reasoning?